docs/es6_Entity.js.html

```diff
@@ -76,10 +76,25 @@ <h1 class="page-title">Source: es6/Entity.js</h1>
 	/**
 	 * Protected: Add a listener to a subentity of this entity.
 	 * @param {*} entityType A entity class that extends this class.
-	 * @param {*} href Href of the entity to be created
+	 * @param {*} href Href or Entity of the entity to be created
 	 * @param {*} onChange callback function that accepts an {entityType} to be called when subentity changes.
 	 */
 	_subEntity(entityType, href, onChange) {
+		// Clean up if that href has already been added.
```

Comment:
`href` is confusing term here at it can mean both href and entity, hard to come up with good name too. `hrefOrEntity` maybe?

Reply:
Yes, I am open to ideas. I also couldn't think of one. Maybe `hrefOrEntity`

Reply:
Maybe this is pointing out a design flaw and I should only have this as a passthrough for the `_subEntityByHref` method?

Reply:
I put that name in the comment then think for a minute, I crossed it out, hmm maybe `source`?

Reply:
or something more general?

Reply:
It might be better to reference it more specifically.  Either embedded subentity, or linked subentity as defined by siren.